UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -580,24 +581,51 @@ public bool IDIsLocalPlayerObject(uint ID)
 
 	[ClientRpc]
 	public void UpdateClientMomentum(Vector3 resetToLocal, Vector2 newMomentum, float inairTime, float inslideTime,
-		int matrixID, float inSpinFactor, bool forceOverride, uint doNotUpdateThisClient)
+		int matrixID, float inSpinFactor, bool forceOverride, uint doNotUpdateThisClient, float TimeSent)
 	{
 		if (isServer) return;
 
 		if (IDIsLocalPlayerObject(doNotUpdateThisClient)) return;
 
+		if (IsFlyingSliding && 0 > (TimeSpentFlying - TimeSent))
```

Comment:
```suggestion
		if (IsFlyingSliding && (TimeSpentFlying - TimeSent) < 0)
```
I think this conveys better the intention and read better at first sight.